meinberlin/templates/email_base.html

```diff
@@ -56,15 +60,26 @@ <h3 style="color: #818181;">
                     </td>
                 </tr>
                 <tr>
-                    <td style="color: #818181; font-size: x-small; padding-top: 15px;">
-                        <p>
-                            {% trans 'meinBerlin is a participation platform operated by' %}<br />
-                            Liquid Democracy e.V., Am Sudhaus 2, D-12053 Berlin
-                        </p>
-                        <p>
-                            <a href="{{ email.get_host }}/contact">{% trans 'Contact' %}</a><br />
-                            <a href="{{ email.get_host }}/terms-of-use">{% trans 'Terms of use' %}</a>
-                        </p>
+                    <td style="padding-top: 15px;">
+                        <table width="100%" border="0" cellpadding="0" cellspacing="0" style="border-collapse: collapse;">
+                            <tr>
+                                {% if organisation.logo %}
+                                    <td style="width: 1%; padding-right: 10px">
+                                        <img width="50" src="cid:logo" style="width: 50px;" alt="MeinBerlin" >
+                                    </td>
+                                {% endif %}
```

Comment:
this also comes from newsletter_base -> maybe do this in every email?

Reply:
Does this show the mB-logo in the footer, when an organisation logo is displayed on top? Then I think, that's a good idea!

Reply:
Yes, thats exactly what it does!